docs/patterns/actions/action-guide-faqs.mdx

```diff
@@ -25,47 +25,12 @@ const tools = await toolset.getTools({actions: ['github_issues_create','github_c
 
 </Accordion>
 
-<Accordion title="How do I find an Action for my Use Case?">
-
-
-<CodeGroup>
-```Python Python Selecting Actions
-use_case="Star a repo on github"
-
-action_enums=tool_set.find_actions_by_use_case(App.GITHUB,use_case=use_case)
-
-tools = tool_set.get_tools(actions=action_enums)
-
-# use tools as per your framework
-```
-
-```Javascript Javascript Selecting Actions
-import { LangchainToolSet } from "composio-core";
-
-const toolset = new LangchainToolSet({
-  entityId: "default",
-});
-
-const actionsList = await toolset.client.actions.list({
-  useCase: "create github issues",
-  apps: "github",
-});
-```
-
-```bash CLI Selecting Actions
-composio actions --use-case 'get channel messages' --app 'slack'
-```
-</CodeGroup>
-
-</Accordion>
-
-
 <Accordion title="Can I filter and get Actions in an App based on tags?">
-
-<Tabs>
-<Tab title="Python">
+Yes, you can filter and get Actions in an App based on tags.
```

Comment:
Great improvement in code formatting and clarity. Consider adding brief explanations for each code example to further enhance understanding for users.

Reply:
yes, added descriptions.